spec/models/appeal_spec.rb

```diff
@@ -1150,6 +1150,41 @@
     end
   end
 
+  describe "#contested_claim?" do
+    subject { appeal.contested_claim? }
+
+    let(:request_issues) do
+      [
+        create(:request_issue, benefit_type: "compensation", nonrating_issue_category: issue_category)
+      ]
+    end
+    let(:appeal) { create(:appeal, request_issues: request_issues) }
+
+    context "when issue category falls under contested claim and category contains string 'Contested Claim'" do
```

Comment:
Awesome that we're thoroughly testing the instances where the method would return true! Since the title of this spec and the next one are pre-pended with the same text, maybe we can wrap the two in a separate context like: 

```suggestion
   context "when issue category falls under contested claim"  do 
      it "returns true if category contains 'Contested Claim" do 
         ...
      end
       it "returns true if category contains 'Apportionment'" do 
         ...
      end
 end
```

This way, all of the truthy specs are nested under the same context and we can easily add more in the future.

Reply:
Ah, this makes sense to group the truth blocks